package.json

```diff
@@ -9,7 +9,7 @@
     "compile": "pnpm -r --filter='./packages/*' run compile",
     "csb-install-FIXME": "pnpm install --no-frozen-lockfile",
     "postinstall": "test -d ./packages/waku/dist || (pnpm -r --filter='./packages/waku' run compile && pnpm install)",
-    "test": "prettier -c . && eslint . && tsc --project . --noEmit",
+    "test": "prettier -c . && eslint . && tsc -b tsconfig.json --diagnostics",
```

Comment:
Curious if we need to pass the default tsconfig?
`tsc -b --diagnostics`

(but, there was `--project .`, so it can't work without passing `tsconfig.json`?)

Reply:
we can